src/rules/utils.ts

```diff
@@ -641,7 +658,8 @@ export const isTestCase = (
       node.callee.object.type === AST_NODE_TYPES.Identifier &&
       TestCaseName.hasOwnProperty(node.callee.object.name) &&
       node.callee.property.type === AST_NODE_TYPES.Identifier &&
-      TestCaseProperty.hasOwnProperty(node.callee.property.name))
+      TestCaseProperty.hasOwnProperty(node.callee.property.name)) ||
+    isConcurrentTestCase(node)
```

Comment:
I think it would be better to bring the checks required from `isConcurrentTestCase` into here, to save on duplicating conditional checks as it should just be a matter of adding a third layer.

Once this PR is merged, I'm going to swing by this method to see about breaking it up a bit and making it more readable.